<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>waiterClass</class>
 <widget class="QMainWindow" name="waiterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>waiter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>70</y>
      <width>601</width>
      <height>192</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <column>
     <property name="text">
      <string>Customer ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Customer Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Table No</string>
     </property>
    </column>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Genrate Customer  Bill</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ReceieveBill</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>618</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="waiter.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>waiterClass</receiver>
   <slot>connetdb()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>waiterClass</receiver>
   <slot>paybill()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>connetdb()</slot>
  <slot>paybill()</slot>
 </slots>
</ui>
